src/vmm/tests/integration_tests.rs

```diff
@@ -178,3 +184,60 @@ fn test_pause_resume_microvm() {
         }
     }
 }
+
+#[test]
+#[cfg(target_arch = "x86_64")]
+fn test_create_snapshot() {
+    let snapshot_path = PathBuf::from("snapshot_file");
```

Comment:
I'd use `tempfile`s for these, just in case an `assert` fails

Reply:
Done.